### ISSUE TITLE
Show Server TPS

### ISSUE THREAD

**Issue by StereoMadnessss** [EwyBoy/FPS-Monitor#7]
The Feature Request is basic but it would be very helpful

**Comment by DrUltraLux** [comment]
Yeah a rough equiv that updates only every 5 minutes or so. This would be a great feature, especially in older versions.
